### PR TITLE
fixes issue with running manifest file without having a workspace

### PR DESCRIPTION
**Pull request by invincibleJai**
Tracks: https://jira.coreos.com/browse/APPAI-368

### CODE REVIEW

**Comment by sunilk747** [comment]
LGTM :+1:

File: src/extension.ts [hunk 1]
```diff
@@ -24,8 +24,10 @@ export let outputChannelDep: any;
 export function activate(context: vscode.ExtensionContext) {
   let disposableFullStack = vscode.commands.registerCommand(
     Commands.TRIGGER_FULL_STACK_ANALYSIS,
-    (uri: vscode.Uri) =>
-      multimanifestmodule.dependencyAnalyticsReportFlow(context, uri)
+    (uri: vscode.Uri) => {
+      let fileUri = uri ? uri : vscode.window.activeTextEditor.document.uri;
```
Comment (arajkumar) [inline src/extension.ts]: nit:
1. Can we use `const` here instead of `let`?
2. Good to have a comment about why `uri` is `null/undefined`.

Comment (invincibleJai) [reply src/extension.ts]: @arajkumar done. ready for review again

**Comment by sunilk747** [comment]
LGTM :+1:

**Pull request merged**